### DISCUSSION TITLE
glTFast Web Demo Unity Project

### DISCUSSION THREAD

**Discussion by JimmyAkl** [atteneder/glTFastWebDemo#10] [Q&A]
Hello, is it possible to share the project / source code used to import on webgl? I am trying to use the glTFast package for webgl to import export gltf / glb on runtime, however it always crashes... Thank you!

**Comment by atteneder** [comment]
The Unity project that was used to build the web binaries can be found [here](https://github.com/atteneder/glTFastDemo/tree/main/projects/glTF-demo-minsize) in this [repository](https://github.com/atteneder/glTFastDemo).

hth
